vmdb/app/models/mixins/file_depot_mixin.rb

```diff
@@ -53,7 +53,7 @@ def requires_credentials?
 
   def validate_depot_credentials
     # This only checks that credentials are present
```

Comment:
At least the code here actually *said* what it expected :smile:   cc @brandondunne